src/main.c

```diff
@@ -228,6 +228,8 @@ static void usage(void)
     "  -p <wildcard>/<flags>  Run developer options for matched AVR devices,\n"
     "                         e.g., -p ATmega328P/s or /S for part definition\n"
     "  -b <baudrate>          Override RS-232 baud rate\n"
+    "  -r                     Open and close (\"touch\") the serial port at\n"
```

Comment:
Perhaps say why this is needed? The manual can (and does) go into the gory details.

\"Touch\" the serial port before programming; needed for XXX, YYY etc bootloader uploads